views/forms/self_referral_form.php

```diff
@@ -41,68 +41,66 @@
                     $self_pers_street_num = $self_address_info[$i];
                 }
             } else {
-                $self_pers_street_name = "";
                 $self_pers_street_name .= $self_address_info[$i] . " ";
             }
         }
 
     $self_pers_zip = !empty($_POST['self_pers_zip']) ? $_POST['self_pers_zip'] : NULL;
     $self_pers_state = !empty($_POST['self_pers_state']) ? $_POST['self_pers_state'] : NULL;
     $self_pers_city = !empty($_POST['self_pers_city']) ? trim($_POST['self_pers_city']) : NULL;
-    $self_apt_info = !empty($_POST['self_apt_info']) ? trim($_POST['self_apt_info']) : NULL;
+    $self_apt_info = !empty($_POST['self_apt_info']) ? trim($_POST['self_apt_info']) : "";
     $self_pers_phone = !empty($_POST['self_pers_phone']) ? $_POST['self_pers_phone'] : NULL;
 
     // Second Card (Additional Information)
-    $self_involvement = !empty($_POST['self_involvement']) ? $_POST['self_involvement'] : NULL;
+    $self_involvement = !empty($_POST['self_involvement']) ? $_POST['self_involvement'] : 0; // TODO: Fix no entry.
         $self_involvement = $self_involvement === "Yes" ? 1 : 0;
-    $self_attended = !empty($_POST['self_attended']) ? $_POST['self_attended'] : NULL;
+    $self_attended = !empty($_POST['self_attended']) ? $_POST['self_attended'] : 0;
         $self_attended = $self_attended === "Yes" ? 1 : 0;
-    $self_ref_source = !empty($_POST['self_ref_source']) ? $_POST['self_ref_source'] : NULL;
-    $reason = !empty($_POST['reason']) ? trim($_POST['reason']) : NULL;
+    $self_ref_source = !empty($_POST['self_ref_source']) ? $_POST['self_ref_source'] : "";
+    $reason = !empty($_POST['reason']) ? trim($_POST['reason']) : "";
 
     // Third Card (Office Information)
     $self_office_firstCall = !empty($_POST['self_office_firstCall']) ? trim($_POST['self_office_firstCall']) : NULL;
     $self_office_returnedCall = !empty($_POST['self_office_returnedCall']) ? trim($_POST['self_office_returnedCall']) : NULL;
     $self_tentative_start = !empty($_POST['self_tentative_start']) ? trim($_POST['self_tentative_start']) : NULL;
     $self_letter_mailed = !empty($_POST['self_letter_mailed']) ? trim($_POST['self_letter_mailed']) : NULL;
-    $self_assigned_to = !empty($_POST['self_assigned_to']) ? trim($_POST['self_assigned_to']) : NULL;
-    $notes = !empty($_POST['notes']) ? trim($_POST['notes']) : NULL;
+    $self_assigned_to = !empty($_POST['self_assigned_to']) ? trim($_POST['self_assigned_to']) : "";
+    $notes = !empty($_POST['notes']) ? trim($_POST['notes']) : "";
     $eID = $_SESSION['employeeid'];
 
     // Testing variables.
 //    $arr = get_defined_vars();
 //    print_r($arr);
```

Comment:
If this commented out code won't be used anymore, I'd remove it so things don't get too cluttered.

Reply:
Got it I'll do that